parl/remote/utils.py

```diff
@@ -52,49 +54,115 @@ def data_process():
     return to_write_lines
 
 
-def load_remote_class(file_name, class_name, end_of_file, in_sys_path):
+def is_class_defined_in_file(cls):
+    """Check whether the class is defined in a python file
+
+    Args:
+        cls: class
     """
```

Comment:
Please assert that the passed argument is a `class`.
https://stackoverflow.com/questions/395735/how-to-check-whether-a-variable-is-a-class-or-not

Reply:
fixed